Export Analysis Reports/Sdl.Community.ExportAnalysisReports/ReportExporterControl.cs

```diff
@@ -445,7 +437,7 @@ private void PrepareProjectToExport(ProjectDetails selectedProject)
 					{
 						// Read sdlproj
 						doc.Load(selectedProject.ProjectPath);
-						_help.LoadReports(doc, selectedProject);
+						_reportService.LoadReports(doc, selectedProject);
```

Comment:
LoadReports was renamed to SetReportInformation() and the selectedProject is needed to be sent in the method because other properties from the object are used within the method _reportService.SetReportInformation(doc, selectedProject)